### ISSUE TITLE
Horizontal overflow

### ISSUE THREAD

**Issue by paulcmal** [Kimeiga/bahunya#1]
Hi, and thank you for building bahunya :heart:! Applying rules to raw elements is a really interesting concept that I've been interested in for years but never really got to do much with.

Anyway, I've noticed on the project homepage that three types of elements can actually overflow horizontally: `pre`, `code` and `table`. This produces the whole page to be horizontally scrollable which is really annoying. Only overflowing elements should be scrollable.

The three elements should have a `overflow-x: auto;` property. Table should be a `display: block` so that overflow properties apply to it. And for some reason applying overflow on `inline-block` code produces some weird vertical misalignment (the element goes up a few pixels compared to the text around)… Simply removing this property seems to do the trick but i don't know if you were counting on it elsewhere.

I didn't have the occasion to do cross-browser/cross-platform review. This was tested on Firefox 60.2.0 ESR on Debian stretch.

**Comment by Kimeiga** [comment]
Thank you so much! I'll hop on it shortly

**Comment by Kimeiga** [comment]
There's a bit of styling that's coming in from the CSS file Prism.js provided to me, so that might be the culprit on `pre` and `code` stuff, but not sure just yet

**Comment by Kimeiga** [comment]
Sorry for forgetting about this issue! I think I've fixed it in this most recent release.

**Comment by paulcmal** [comment]
Sorry to bring bad news, but the homepage still overflows because of the table. It seems that `display: table` (at least in Firefox) does overflow by default unless you wrap it in a div. I don't know if the table display spec is meant as such, or if that's a browser implementation bug.

The only semantic workaround i found was [nice Flexbox hacks](https://stackoverflow.com/questions/14834198/table-scroll-with-html-and-css/22801367#22801367), in case that inspires you :)

**Comment by Kimeiga** [comment]
HEY! I apologize for the lack of communication but I'm basically almost done revamping the entire framework to be lighter, have a more internally consistent codebase, and basically look a lot better. When I release i'll let you know.

I think this is related to the other issue with the table, and since I redid the tables from the ground up on this second pass I think it should no longer be wider than the body or any smaller containers it is put in. but thank you for the flexbox link!

I will probably finish this after finals week so basically this summer so sorry for the further delay!

**Comment by Kimeiga** [comment]
Hi Paul! 

Well it took 4 years and redoing the whole css framework to fix it and make it look nicer but I made sure
1. `<tables>`s overflow nicely on small screens
2. `<pre>`s overflow nicely even if there is no `<code>` within them
3. `<pre><code>`s overflow nicely

You'll have to drop in the new CDN link as I have changed the folder structure quite a lot.

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kimeiga/bahunya/dist/bahunya.min.css">
```

Cheers and thanks for making an issue so long ago!
